From b79e590733a55cc621c5ab723012e0455db28a83 Mon Sep 17 00:00:00 2001
From: Liu Ying <Ying.liu@freescale.com>
Date: Wed, 12 Sep 2012 09:36:12 +0800
Subject: ENGR00223797-5 MX6 SabreSD:Initialize bypass reset for IPUv3 pdata

This patch initializes bypass reset field to false for IPUv3 pdata,
so we don't support bypass reset by default unless we change them
to true in other places specificly.

Signed-off-by: Liu Ying <Ying.Liu@freescale.com>
(cherry picked from commit 6c294588a1987810ed4d45bcb5e1d3416d66d0be)
---
 arch/arm/mach-mx6/board-mx6q_sabresd.c |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm/mach-mx6/board-mx6q_sabresd.c b/arch/arm/mach-mx6/board-mx6q_sabresd.c
index e198617..3aabc75 100644
--- a/arch/arm/mach-mx6/board-mx6q_sabresd.c
+++ b/arch/arm/mach-mx6/board-mx6q_sabresd.c
@@ -1358,9 +1358,11 @@ static struct imx_ipuv3_platform_data ipu_data[] = {
 	{
 	.rev = 4,
 	.csi_clk[0] = "clko_clk",
+	.bypass_reset = false,
 	}, {
 	.rev = 4,
 	.csi_clk[0] = "clko_clk",
+	.bypass_reset = false,
 	},
 };
 
-- 
1.7.9.5

